subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -598,6 +600,23 @@ Set JVM arguments.
 `-Dorg.gradle.java.home`::
 Set JDK home dir.
 
+[[sec:task_options]]
+== Task options
```

Comment:
This should somehow link to https://docs.gradle.org/current/userguide/custom_tasks.html#sec:declaring_and_using_command_line_options